.circleci/config.yml

```diff
@@ -290,6 +290,22 @@ jobs:
             PATH="venv/bin:$PATH" ./script/generate_sql \
               --sql-dir /tmp/workspace/generated-sql/sql/ \
               --target-project moz-fx-data-shared-prod
+
+            # generate all of the relevant incremental SQL for GLAM
+            products="
+              org_mozilla_fenix_glam_nightly
+              org_mozilla_fenix_glam_beta
+              org_mozilla_fenix_glam_release
+            "
+
+            for product in $products; do
+              PATH="venv/bin:$PATH" \
+              STAGE="incremental" \
+              DST_PROJECT="moz-fx-data-glam-prod-fca7" \
+              PRODUCT="$product" \
+              ./script/glam/generate_glean_sql
+            done
+
```

Comment:
I think it would be most appropriate to move this block into `./script/generate_sql` right before the final `bqetl dependency record` call. That will make sure it also runs for the `private-generated-sql` branch and for local testing.

Reply:
The `generate_sql` script takes in a target project parameter which will end up being different for this set of ETL (moz-fx-data-glam-prod-fca7). Should I stick this in a conditional block?

Reply:
The `target-project` param is really only potentially useful for switching between shared-nonprod and shared-prod. And I suspect it doesn't fully work for shared-nonprod, since we don't have the dry run service for nonprod. It's definitely confusing, and we should document that the param is limited.

Given that, I think it's fine to have this invocation hard code the `moz-fx-data-glam-prod-fca7` project and add a comment about it.